server/tools/docker-entrypoint.sh

```diff
@@ -229,6 +233,19 @@ fi
 /opt/jboss/tools/autorun.sh
 /opt/jboss/tools/vault.sh
 
+##################################
+# Print first login and password #
+##################################
+
+echo "========================================================================="
+echo ""
+echo "  Your first login and password:"
+echo "  Login: ${DEFAULT_USER}"
```

Comment:
Replace ```DEFAULT_USER``` / ```DEFAULT_PASSWORD``` below with their ```KEYCLOAK_USER``` / ```KEYCLOAK_PASSWORD``` counterparts. This way, it would just reprint the username / password credentials combination for the admin user, if the variables were specified. Or display the autogenerated / actual password.